CityOfDetroit/bloom
# Add hero image to homepage

# Pull Request Template

## Issue

Addresses #343 


## Description

Added Detroit skyline image to homepage.
![image](https://user-images.githubusercontent.com/82653098/128871117-1a422d1c-634b-43d2-a614-6fdf670782ca.png)

## Type of change


## How Can This Be Tested/Reviewed?

Visit the homepage and notice the hero image.  Resize your screen for both desktop and mobile resolutions.


## Checklist:
